src/System.Net.Mail/src/System/Net/Mail/SmtpClient.cs

```diff
@@ -404,7 +404,7 @@ internal MailWriter GetFileMailWriter(string pickupDirectory)
             }
 
             FileStream fileStream = new FileStream(pathAndFilename, FileMode.CreateNew);
-            return new MailWriter(fileStream);
+            return new MailWriter(fileStream, encodeForTransport: false);
```

Comment:
This is essentially the fix, in that we're now passing false instead of true here, right?

Reply:
Yes